docker-nlp-toolset/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM maluuba/tomcat7
+MAINTAINER Maluuba NLP Team <nlp@maluuba.com>
+
+# Git
+RUN apt-get install -qqy git
+
+# Maven
+RUN apt-get remove maven2
+ENV MAVEN_VERSION 3.2.5
+RUN curl -sSL http://archive.apache.org/dist/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz | tar xzf - -C /usr/share \
+  && mv /usr/share/apache-maven-$MAVEN_VERSION /usr/share/maven \
+  && ln -s /usr/share/maven/bin/mvn /usr/bin/mvn
+ENV MAVEN_HOME /usr/share/maven
+
+# Ruby, JRuby, and RVM
+# Default to production, as per the README, this can be overriden in `docker run` by adding "-e RAILS_ENV=development"
+ENV RAILS_ENV production 
+RUN gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
+RUN curl -sSL get.rvm.io | bash -s stable
+RUN bash -l -c "rvm requirements"
```

Comment:
Looks like my comments regarding the `bash -l -c` disappeared, any ideas @usmanismail